packages/concerto-core/test/data/decoratorcommands/map-declaration.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$class" : "org.accordproject.decoratorcommands@0.3.0.DecoratorCommandSet",
+    "name" : "web",
+    "version": "1.0.0",
+    "commands" : [
+        {
+            "$class" : "org.accordproject.decoratorcommands@0.3.0.Command",
```

Comment:
When you bump `org.accordproject.decoratorcommands` version you will have to find/replace in this file...